pages/common/pixterm.md

```diff
@@ -0,0 +1,21 @@
+# pixterm
+
+> Image printing in the terminal.
+> See also: `chafa`, `catimg`.
+> More information: <https://github.com/eliukblau/pixterm>.
+
+- Print a static image to the terminal:
+
+`pixterm {{path/to/file}}`
+
+- Use the image's original aspect ratio:
+
+`pixterm -s 2 {{path/to/file}}`
+
+- Specify a custom aspect ratio with [t]erminal [r]ows and [c]olumns:
```

Comment:
```suggestion
- Specify a custom aspect ratio using a specific number of [t]erminal [r]ows and [c]olumns:
```

I think this makes more sense to anyone reading it, although it is a bit longer.